### ISSUE TITLE
Label matching, 182 or 183 labels?

### ISSUE THREAD

**Issue by AlejandroHernandezMunuera** [nightrome/cocostuff#8]
Hi there,

I'm trying to use your model and migrate it to tensorflow using the caffe-tensorflow repository and your caffemodel and prototxt files. The problem I face now is that the Deeplab VGG-16 model trained on COCO-Stuff that you offer on thid repository only outputs 182 different labels but as I understand it should return 183 (91 for COCO, 91 for stuff and 1 for unlabeled).

Please let me know if I'm missing something,
thank you in advance

**Comment by nightrome** [comment]
Hi. That is correct. We do not output predictions for the unlabeled class, as it is (arguably) not a semantic class. Similar to PASCAL VOC it is treated as a void class and ignored from the loss. If you desperately need it you can retrain the network using the provided code or maybe you could just assign the `unlabeled` category to every pixel with a low confidence for all other classes.
